### PR TITLE
Allow the HoP to give custom titles their own categories

### PR DESCRIPTION
**Pull request by ShiftyRail**
[content]
<s>It *should* work but when I tried to test it the Identification Computer UI bugged out on me, just not displaying anything
Which I fail to understand why as the only change I did before and after the UI bugged was to add an extra pair of parenthesis around the conditions in `obj.dm`

It's not even the Crew Manifest(s) in general that are breaking because the PDA one and the verb work properly </s>

Tested.

:cl:
- tweak: Allow the HoP to give custom titles their own categories

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Tested in on test serb as I had some UI issues on local.
Works fine.

File: code/game/machinery/computer/card.dm [hunk 1]
```diff
@@ -264,6 +264,15 @@
 						//let custom jobs function as an impromptu alt title, mainly for sechuds
 						if(temp_t && modify)
 							modify.assignment = temp_t
+					if (!(temp_t in all_jobs_txt))
+						var/new_dept = input("Choose the departement this job belongs to.") as null|anything in departement_list
+						if (new_dept)
+							for (var/list/L in list(data_core.general, data_core.medical, data_core.security,data_core.locked))
+								if (L)
```
Comment (ShiftyRail) [inline code/game/machinery/computer/card.dm]: I copypasted it from another proc.
This shouldn't ever be false, so it's probably better to put `if (L.len)`

**Comment by SonixApache** [comment]
What exactly does this do?

**Comment by Dacendeth** [comment]
@SonixApache I think it lets you make a custom title and have it fall under the security/medical/etc categories, instead of always misc?

**Comment by SonixApache** [comment]
Oh, like that. But it'll still show a [?] on the secHUDs unless it's set to a specific access type, right?

**Comment by ShiftyRail** [comment]
Razz is right.
I don't think they'll show up as a ? on the secHuD unless the access was manually set and it's a blank ID card ; as far as I know, there's a "hidden rank" variable on your ID that retains the last "real rank" your ID had.

**Pull request merged**